tests/e2e/vms/xsvm.go

```diff
@@ -226,40 +226,46 @@ var _ = ginkgo.Describe("[XSVM]", ginkgo.Label("xsvm"), func() {
 
 		// Stream pings to the server and block until all events are received
 		// back.
-		wg := &sync.WaitGroup{}
-		wg.Add(2)
+		eg := &errgroup.Group{}
 
 		n := 10
-		go func() {
-			defer func() {
-				ginkgo.GinkgoRecover()
-				wg.Done()
-			}()
+		eg.Go(func() error {
+			defer ginkgo.GinkgoRecover()
 
 			for i := 0; i < n; i++ {
 				msg := fmt.Sprintf("ping-%d", i)
-				require.NoError(stream.Send(&xsvm.StreamPingRequest{
+				if err := stream.Send(&xsvm.StreamPingRequest{
 					Message: msg,
-				}))
+				}); err != nil {
+					return err
+				}
+
 				log.Info("sent message", zap.String("msg", msg))
 			}
-		}()
 
-		go func() {
-			defer func() {
-				ginkgo.GinkgoRecover()
-				wg.Done()
-			}()
+			return nil
+		})
+
+		eg.Go(func() error {
+			defer ginkgo.GinkgoRecover()
```

Comment:
Do we still need to call `ginkgo.GinkgoRecover()` since we are no longer calling `require` in this goroutine?